src/spec/integration/product-type-importer.spec.coffee

```diff
@@ -70,8 +70,8 @@ describe 'ProductTypeImporter', ->
       sphereClient = new SphereClient options
 
   beforeEach ->
-    sphereClient.productTypes.fetch()
-    .then (res) ->
+    sphereClient.productTypes
+    .process (res) ->
```

Comment:
@butenkor default per page is 20, so `res` will only contain 20 entries. `Promise.map` would trim this down further, but I am not sure we really need to go below 20 here.